pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = [
+    "setuptools<60.0.0",
+    "wheel",
+]
```

Comment:
We really need very little to install `pykokkos`, and it is lightning fast, so no huge advantage to wheels just yet I suppose.